cppcache/src/ExceptionTypes.cpp

```diff
@@ -297,7 +297,25 @@ const std::string& getThreadLocalExceptionMessage();
       PutAllPartialResultException ex(message);
       throw ex;
     }
-    default: {
+    case GF_NOERR:
```

Comment:
Same as above - adding a zillion case statements doesn't make code clearer.

Reply:
Let me know what you think of the change @pdxcodemonkey .  I have mixed feelings about it...

Reply:
![pdxcodemonkey?](http://media2.giphy.com/gifsu/1qbj0e6kDAjS7elwu0/giphy-caption.gif?cid=6104955e5f2b035851577a4b4d4cc90a)

(I mostly just wanted to post that gif, no particular rush 😃)